include/merlin_hashtable.cuh

```diff
@@ -889,19 +890,31 @@ class HashTable {
 
     reader_shared_lock lock(mutex_);
 
+    uint32_t value_size = options_.dim * sizeof(V);
     if (is_fast_mode()) {
-      using Selector =
-          SelectLookupKernelWithIO<key_type, value_type, score_type>;
-      static thread_local int step_counter = 0;
-      static thread_local float load_factor = 0.0;
-
-      if (((step_counter++) % kernel_select_interval_) == 0) {
-        load_factor = fast_load_factor(0, stream, false);
+      // Only bucket_size = 128
+      // On A100, the maximum dim which Pipeline support is 224 floats
+      if (options_.max_bucket_size == 128 &&
+          value_size <= (224 * sizeof(float))) {
```

Comment:
We'd better avoid the magic number, and make it a private member of `HashTable` or better form. If the 224 depends on the GPU hardware setting, we need to calculate it at initialize phrase.

Reply:
OK, I will make arch infomation as tempate type to select the kernel related config at compile time.